package.json

```diff
@@ -9,7 +9,8 @@
     "start": "NODE_OPTIONS='--trace-warnings' next start -p $PORT --keepAliveTimeout 120000",
     "lint": "standard",
     "test": "NODE_OPTIONS='--experimental-vm-modules' jest",
-    "worker": "tsx --trace-warnings worker/index.js"
+    "worker": "tsx --trace-warnings worker/index.js",
+    "worker:dev": "tsx --trace-warnings --watch worker/index.js"
```

Comment:
Interesting, this fortunately restarts the worker if any file in worker/ was changed.